staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2446,6 +2446,22 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container.
+	// This may only be set for init containers.
+	// The only supported value is "Always".
+	// When set to "Always", this init container will continuously restart if it
+	// exits for any reason, until all the regular containers have terminated.
```

Comment:
Simplify this sentence.

“Always results in this container being continually restarted on exit until all regular containers have terminated.”

Then add additional details

Reply:
Fixed